communication.rst

```diff
@@ -92,28 +92,127 @@ RSS feed readers.
 .. _web gateway: https://mail.python.org/archives/
 
 
-Discourse (discuss.python.org web forum)
-----------------------------------------
+Discourse
+-----------
+**(discuss.python.org web forum)**
 
 We have our own `Discourse`_ forum for both developers and users. This forum
 complements the `python-dev`_, `python-ideas`_, `python-help`_, and
 `python-list`_ mailing lists.
 
-Most core development discussions take place in the open forum categories for
-`PEPs`_ and `Core Development`_. (These are the Discourse equivalents to the
-python-dev mailing list)
-
-The `Committers`_ category restricts posting to core developers only, and is
-used more for announcements and notifications, rather than for discussions. It
-is also the venue for core developer promotion votes. (This category is the
-equivalent of the python-committers mailing list)
-
-There is a final, rarely used, core development category on Discourse that is
-only visible to core developers. This can be used to share administrative
-information with all core developers in a non-public forum (e.g. logistics for
-in person core development sprints), as well as for individual core developers
-to share info that they'd like other core devs to be aware of, but would prefer
-not to have permanently archived on the internet.
+This forum has many different categories and most core development discussions
+take place in the open forum categories for `PEPs`_ and `Core Development`_.
+(These are the Discourse equivalents to the python-dev mailing list)
+Most categories are open for all users to read and post with the exception of
+Committers and Core Development categories.
```

Comment:
Core Dev is open for everyone, no?

Reply:
My bad. I was just editing the previous content presuming that category holds similar situations as documented. Seems like it has been changed.